openslides/utils/class_register.py

```diff
@@ -0,0 +1,39 @@
+class RegisterSubclasses(type):
+    """
+    Metaclass to track subclasses of a class.
+
+    The metaclass adds an attribute 'subclasses' to the metaclass. This
+    attribute is a set containing all derivated classes.
+
+    After a new subclass is created, the classmethod on_subclass_created is
+    called.
+
+    Example:
+    class Base(metaclass=RegisterSubclasses):
+        pass
+
+    class Child(Base):
+        pass
+
+    assert Child in Base.subclasses
```

Comment:
I think this would be True: `Child.subclasses is Base.subclasses`.

If I am right this looks missleading.